packages/framework/tests/Unit/Component/UploadedFile/UploadedFileLocatorTest.php

```diff
@@ -105,10 +92,26 @@ public function testGetUploadedFileUrl()
         $uploadedFileMock->method('getFilename')->willReturn('dummy.txt');
         $uploadedFileMock->method('getEntityName')->willReturn('entityName');
 
-        $uploadedFileLocator = new UploadedFileLocator($uploadedFileDir, $uploadedFileUrlPrefix);
+        $uploadedFileLocator = $this->createUploadedFileLocator('/assets/');
         $this->assertSame(
             'http://www.example.com/assets/entityName/dummy.txt',
             $uploadedFileLocator->getUploadedFileUrl($domainConfig, $uploadedFileMock)
         );
     }
+
+    /**
+     * @param $uploadedFileDir
+     * @param $uploadedFileUrlPrefix
+     * @return \Shopsys\FrameworkBundle\Component\UploadedFile\UploadedFileLocator
+     */
+    private function createUploadedFileLocator($uploadedFileUrlPrefix = '', $has = true)
+    {
+        $uploadedFileDir = __DIR__ . '/UploadedFileLocatorData/';
+        $filesystemMock = $this->getMockBuilder(FilesystemInterface::class)
```

Comment:
This could be written in a shorter way `$this->createMock(FilesystemInterface::class)`